app/controllers/jobseekers/confirmations_controller.rb

```diff
@@ -1,15 +1,15 @@
 class Jobseekers::ConfirmationsController < Devise::ConfirmationsController
   def show
-    super if request.method == "POST"
+    super
   end
```

Comment:
Question :question: 
Now that `#show` does not override any logic from the parent, could we remove this definition? Or is needed?

Reply:
Ah yep, I'm we can just remove it 👍